spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe FoodFacade do 
+  it 'returns food search data' do 
+    search = FoodFacade.search_foods('sweet potatoes')
+
+    expect(search).to be_a FoodSearchResult
+    expect(search.total_hits).to be_a Integer
+    expect(search.foods).to be_a Array
+    expect(search.foods.first).to be_a Food
+  end 
+end
```

Comment:
You can take a look at the [all](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/all-matcher) method to check all elements in an array. (Referring to line 10)

Reply:
You'll also want a quick test for the `keyword` method.